### ISSUE TITLE
dbplyr 2.4.0- breaking changes

### ISSUE THREAD

**Issue by kelseyroberts** [tidyverse/dbplyr#1416]
With the updates to dbplyr version 2.4.0 it seems like some of the arguments that worked in the previous version, I was using version 2.3.2,  no longer work. 

The 3 main differences we are seeing is 
1. The "database" argument needs to be called “catalog” now
2. "schemaname" needs to be called “schema” 
3. "tablename" needs to be called “table”

---

Database to catalog 
For example this call used to work with the argument “database”:

```r
database_connection_object%>%
  tbl(DBI::Id(
    database = "db1",
    schema = "schema1",
    table = "table1"
  )) %>%
  collect()
```
This now requires “database” to be named “catalog” as seen below:
```r
database_connection_object%>%
  tbl(DBI::Id(
    catalog = "db1",
    schema = "schema1",
    table = "table1"
  )) %>%
  collect()
```

Schemaname and Tablename
“schemaname” and “tablename” were arguments that worked before and are now also giving errors: 
```r
database_connection_object%>%
  tbl(DBI::Id(
    database = "db1",
    schemaname = "schema1",
    tablename = "table1"
  )) %>%
  collect()
```

schemaname and tablename now are required to be “schema” and “table”:
```r
database_connection_object%>%
  tbl(DBI::Id(
    database = "db1",
    schema= "schema1",
    table= "table1"
  )) %>%
  collect()
```

**Comment by hadley** [comment]
Yes, sorry about that. I'm working on a fix in #1396. I'm hoping I can finish it off next week and get it to CRAN soon after.

**Comment by hadley** [comment]
FWIW the goal is to get to a place where you can write:

```R
con %>%
  tbl(I("db1.schema1.table1")) %>%
  collect()
```

Which is quite a bit less typing 😄
